src/core_plugins/kibana/public/dashboard/panel/dashboard_panel.test.js

```diff
@@ -2,34 +2,40 @@ import React from 'react';
 import _ from 'lodash';
 import { mount } from 'enzyme';
 import { DashboardPanel } from './dashboard_panel';
+import { DashboardViewMode } from '../dashboard_view_mode';
 import { PanelError } from '../panel/panel_error';
+import { store } from '../../store';
```

Comment:
Should these "integration tests" create a new store per test (or test file) instead of reusing shared state? Might be okey to re-use it, but it feels like it can cause flaky tests over time.

Reply:
I'd be open to this but I'm not sure how to do it without exposing the createStore function purely for the sake of tests, and it doesn't feel right to change code just for testing purposes.  Unless there is an easy way to "reset" the state that I'm not aware of.